plugins/copr.py

```diff
@@ -471,6 +471,8 @@ def _guess_chroot(self):
                 chroot = ("fedora-rawhide-" + distarch)
             else:
                 chroot = ("fedora-{0}-{1}".format(dist[1], distarch))
+        elif "CentOS Stream" in dist:
+            chroot = ("centos-stream-{}-{}".format(dist[1].split(".", 1)[0], distarch if distarch else "x86_64"))
```

Comment:
Thank you for the update.

This is not something I'd love to make decisions on.  I'd suggest you: ask the CentOS Stream community.

Just note that the CentOS Stream build chroots have [a lower chance to even exist in the project](https://copr.fedorainfracloud.org/status/stats/):
![image](https://github.com/rpm-software-management/dnf-plugins-core/assets/3277839/e45178a9-5283-456b-ab66-eb85f53ae6e0)

And then, from my observation only, it is a relatively known fact that `epel-*` is the default decision DNF copr plugin on the Enterprise Linux-based distros, so if anyone has to provide some "stable and supported content" in Copr for EL, they provide `epel-` chroots. The `centos-stream-*` or `centos-stream+epel-*` chroots in Fedora Copr are typically used just for CI testing.